.github/ISSUE_TEMPLATE/app_rendering_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Card Rendering Bug Report
```

Comment:
Maybe capitals at the beginning? "Name", "Title", ...
Unless those are case-sensitive keywords of course.

Reply:
I'm following the GitHub convention for YAML files where fields are all lowercase. YAML files typically use all lowercase with underscores separating words....capitals are not typically used in YAML